From 159ed4643b151740e0ec7d16d947b867a97662e0 Mon Sep 17 00:00:00 2001
From: Denis Prokhorenko <depofficial@mail.ru>
Date: Wed, 14 Oct 2020 22:02:09 +0300
Subject: [PATCH] modify two lines in firstFile.txt

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index d3d1712..1dc5f5f 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,5 +1,5 @@
-1
-1
+11
+11
 1
 1
 1
-- 
2.26.2.windows.1

